looks like there's one more unit test to fix for that helper function, otherwise looks great!:
```
  879 passing (1m)
  18 pending
  1 failing

  1) profile helpers:
       normalizeSubmissionStatus
         should normalize "received" from api submission status:

      AssertionError: expected 'inProgress' to equal 'received'
      + expected - actual

      -inProgress
      +received
      
      at forEach (src/applications/personalization/dashboard/tests/helpers.unit.spec.js:44:53)
      at Array.forEach (<anonymous>)
      at Context.<anonymous> (src/applications/personalization/dashboard/tests/helpers.unit.spec.js:43:53)
      at Test.Runnable.run (node_modules/mocha-snapshots/src/index.js:19:22)
      at processImmediate (internal/timers.js:462:21)
      at process.callbackTrampoline (internal/async_hooks.js:134:14)
```

--- a/src/applications/personalization/dashboard/tests/helpers.unit.spec.js
+++ b/src/applications/personalization/dashboard/tests/helpers.unit.spec.js
@@ -40,15 +40,17 @@ describe('profile helpers:', () => {
     });
 
     it('should normalize "received" from api submission status', () => {
-      ['received', 'processing', 'success', 'VBMS'].forEach(value => {
+      ['VBMS'].forEach(value => {
         expect(normalizeSubmissionStatus(value)).to.equal('received');
       });
     });
 
     it('should normalize "inProgress" from api submission status', () => {
-      ['pending', 'uploaded'].forEach(value => {
-        expect(normalizeSubmissionStatus(value)).to.equal('inProgress');
-      });
+      ['pending', 'uploaded', 'received', 'processing', 'success'].forEach(
+        value => {
+          expect(normalizeSubmissionStatus(value)).to.equal('inProgress');
+        },
+      );
     });
   });
 

